docker/hailgenetics/hail/Dockerfile

```diff
@@ -1,32 +1,16 @@
 FROM {{ hail_ubuntu_image.image }}
 
-ENV MAKEFLAGS -j2
-
 RUN hail-apt-get-install \
     curl \
     git \
     jq \
     liblapack3 \
     openjdk-8-jre-headless \
     rsync \
-    unzip bzip2 zip tar tabix lz4 \
-    vim pv
-
-# source: https://cloud.google.com/storage/docs/gsutil_install#linux
-RUN curl https://sdk.cloud.google.com | bash && \
-    find \
-      /root/google-cloud-sdk/bin/ \
-      -type f -executable \
-    | xargs -I % /bin/sh -c 'set -ex ; ln -s ${PWD}/% /usr/local/bin/$(basename %)'
+    vim
 
 COPY hail/python/pinned-requirements.txt requirements.txt
-RUN hail-pip-install -r requirements.txt \
-      ipython \
-      matplotlib \
-      numpy \
-      scikit-learn \
-      dill \
-      scipy
```

Comment:
The only ones that aren't picked up in requirements.txt are scikit-learn and matplotlib. Scikit-learn is kinda specialized, but I could be convinced to keep it. Matplotlib is similar in size but we already provide two plotting libraries: plotly and bokeh (bokeh is like 50MiB).